### PR TITLE
Verbose testing functionality

### PR DESCRIPTION
**Pull request by safinsingh**
Adds testing functionality (this is from a while ago, but I've updated it and I think it'd be a good addition to continuous integration)

### CODE REVIEW

File: cmd/checks.go [hunk 1]
```diff
@@ -230,7 +231,14 @@ func fileContainsRegex(fileName, expressionString string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	matched, err := regexp.Match(expressionString, []byte(fileContent))
+	matched := false
+	found := false
+	for _, line := range strings.Split(fileContent, "\n") {
+		found, err = regexp.Match(expressionString, []byte(line))
+		if found {
+			matched = found
```
Comment (sourque) [inline cmd/checks.go:238]: Could just have one variable for found, then break if found right?

File: cmd/checks.go [hunk 2]
```diff
@@ -272,12 +286,12 @@ func dirContainsRegex(dirName, expressionString string) (bool, error) {
 // fileEquals calculates the SHA1 sum of a file and compares it
 // with the hash provided in the check
 func fileEquals(fileName, fileHash string) (bool, error) {
-	fileContent, err := readFile(fileName)
+	fileContent, err := ioutil.ReadFile(fileName)
```
Comment (sourque) [inline cmd/checks.go]: Any reason to not use our readFile wrapper?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -22,4 +26,4 @@ jobs:
           go get -v -t -d ./...
 
       - name: Test
-        run: go test -v .
+        run: cd cmd && go test -v .
```
Comment (sourque) [inline .github/workflows/test.yml]: Can also `go test -v ./cmd` I think?

Comment (safinsingh) [reply .github/workflows/test.yml]: Yeah makes more sense

File: cmd/checks.go (hunk 2 above)
Comment (safinsingh) [reply cmd/checks.go]: Hmmm let me change that and see what the CI says

**Review by sourque — APPROVED** [review]

**Pull request merged**